dds2val/OverallSequence.puml

```diff
@@ -0,0 +1,23 @@
+@startuml OverallSequence
+title DDS on SDV
+box "Container-1"
+participant databroker
+end box
+databroker <-- ddsprovider : grpc_connect
+alt on connection
+ddsprovider --> databroker : register_datapoints
+ddsprovider --> DDS_network : subscribe to topics
+box "Container-2"
+participant ddspublisher1
+end box
+box "Container-3"
+participant ddspublisher2
+end box
+ddspublisher1 --> DDS_network : publish dds message
+ddspublisher2 --> DDS_network : publish dds message
+alt on data reception
+ddsprovider --> databroker : update_datapoint
+end note
+end
+@enduml
+# was converted to Mermaid diagram, because Github can dynamically load mermaid diagrams in README
```

Comment:
But then we can remove this file right, as content is directly included in README?